rsign/request.py

```diff
@@ -14,7 +16,7 @@ def normalize(timestamp, nonce, method, path, host, port):  # pylint: disable=R0
     """ Accepts args as strings and returns the normalized form for signing """
     normalized = '\n'.join([
```

Comment:
Here we make a string and decode nonce (which might be bytes, can it?). I think the only usage of normalized is in sign_string which requires bytes. If my understanding is correct, should not we just produce bytes here?